### DISCUSSION TITLE
为 bridge 开启浏览器转发导致 panic，请问这是正常的，还是一个 bug？

### DISCUSSION THREAD

**Discussion by ghost** [v2fly/v2ray-core#1766] [Q&A] [answered]
如题，这里的 bridge 指反代功能中帮助 bridge 一方发送实际流量的 websocket 出站。测试中我在该出站和常规代理用途出站使用了相同的配置（即以相同的协议和设置连接 VPS），差异仅在于二者的 tag 及引用这些 tag 的路由规则（同文档，不开启浏览器转发时使用无误）。在此基础上，调整两个出站中 useBrowserForwarding 的值尝试启动 V2Ray，得到的结果是：只要开启 bridge 所用出站的浏览器转发就会报 panic: V is not in context.，不论另一出站是否开启浏览器转发；只为后者开启浏览器转发时 V2Ray 正常启动。

**Comment by xiaokangwang** [comment, marked as answer]
这个其实是一个bug，但是暂时还没来及修。。。。

**Reply by ghost** [reply]
感谢回复！！只是写启动脚本时碰巧发现了，没有刚需。感谢回复！！！！！！
